### PR TITLE
Corrected grammar and mispelling

### PR DESCRIPTION
**Pull request by zallaevan**
##### Summary

Corrected some grammar mistakes in the metadata file, plus amending the misspelling of the perf command.

##### Test Plan

N/A

##### Additional Information

N/A

### CODE REVIEW

**Review by ilyam8 — APPROVED** [review]

**Comment by zallaevan** [comment]
Sorry, I messed up. It is my first pull request, so once approved I thought that I needed to close this. Instead, I need to first merge it to the master branch, innit?

**Comment by ilyam8** [comment]
Thanks for fixing misspellings, @zallaevan!

File: src/collectors/perf.plugin/metadata.yaml [hunk 1]
```diff
@@ -23,14 +23,14 @@ modules:
...
-        method_description: "It uses syscall (2) to open a file descriptior to monitor the perf events."
+        method_description: "It uses syscall (2) to open a file descriptor to monitor the perf events."
       supported_platforms:
         include:
           - Linux
         exclude: []
       multi_instance: true
       additional_permissions:
-        description: "It needs setuid to use necessary syscall to collect perf events. Netada sets the permission during installation time."
+        description: "It needs setuid to use the necessary syscall to collect perf events. Netdada sets the permission during installation time."
```
Comment (ilyam8) [inline src/collectors/perf.plugin/metadata.yaml:33]: > Netdada

Netdata

I missed that during the review. Fixed in #17702

Comment (zallaevan) [reply src/collectors/perf.plugin/metadata.yaml:33]: Mistake in the correction lmao I need to focus more next time!

**Pull request merged**